providers/airbyte/docs/changelog.rst

```diff
@@ -27,6 +27,24 @@ Changelog
 ---------
 
 
+5.1.0
+.....
+
+.. note::
+    This release of provider is only available for Airflow X.Y+ as explained in the
```

Comment:
Concrete version number to be added? I assume 2.10.x?

Reply:
...and I assume same yields for other provider changelogs as well?

Reply:
Yes I didn't change it yet. Will bulk do it in the next commit.

Reply:
Same question :) . I just review changelogs now - that might be a hint for others - all the other things are auto-generated by chanelogs are important to take a look at :)